extensions/amp-story/1.0/page-advancement.js

```diff
@@ -426,6 +426,21 @@ class ManualAdvancement extends AdvancementConfig {
     });
   }
 
+  /**
+   * Checks if current element is valid to trigger a tooltip.
+   * @param {!Event} event
+   */
+  isTooltipValidDescendant_(event) {
+    let valid = true;
+    return !!closest(dev().assertElement(event.target), el => {
+      if (el.tagName.toLowerCase() == 'amp-story-cta-layer') {
+        valid = false;
+        return false;
+      }
+      return el.tagName.toLowerCase() == 'amp-story-page' && valid;
+    });
```

Comment:
Maybe consider passing `this.element_` as a third parameter to stop traversing the DOM once we reach `<amp-story>`?

Reply:
sure, it's important to note `this.element_` could also be an `AmpStoryPage` but that is probably fine.